### ISSUE TITLE
Error in make test with -> ./bin/build version uberjar && yarn run test

### ISSUE THREAD

**Issue by jonatasfreitasv** [metabase/metabase#6086]
Metabase 0.26.1

Without any modification, original master branch.

```bash
 RUNS  frontend/test/components/Logs.unit.spec.js

 PASS  frontend/test/components/Logs.unit.spec.js

Summary of all failing tests
 FAIL  frontend/test/visualizations/components/LineAreaBarRenderer.unit.spec.js (24.583s)
  ● LineAreaBarRenderer › should display hourly data (in the browser's timezone) in X axis and tooltip consistently and correctly

    expect(received).toEqual(expected)
    
    Expected value to equal:
      ["1 AM - January 1, 2016", "4 AM - January 1, 2016"]
    Received:
      Array []
    
    Difference:
    
    - Expected
    + Received
    
    -Array [
    -  "1 AM - January 1, 2016",
    -  "4 AM - January 1, 2016",
    -]
    +Array []
      
      at onHoverChange (frontend/test/visualizations/components/LineAreaBarRenderer.unit.spec.js:114:101)
      at frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js:1495:116
      at HTMLUnknownElement.<anonymous> (frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js:1398:585)
      at HTMLUnknownElement.__onmousemove (node_modules/d3/d3.js:1120:18)
      at invokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:219:27)
      at HTMLUnknownElementImpl._dispatch (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:126:9)
      at HTMLUnknownElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:87:17)
      at HTMLUnknownElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/nodes/HTMLElement-impl.js:36:27)
      at HTMLUnknownElement.dispatchEvent (node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:61:35)
      at dispatchUIEvent (frontend/test/visualizations/__support__/visualizations.js:92:13)
      at frontend/test/visualizations/components/LineAreaBarRenderer.unit.spec.js:128:49
          at Promise (<anonymous>)
      at Object.<anonymous> (frontend/test/visualizations/components/LineAreaBarRenderer.unit.spec.js:93:16)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)


Test Suites: 1 failed, 69 passed, 70 total
Tests:       1 failed, 56 skipped, 586 passed, 643 total
Snapshots:   17 passed, 17 total
Time:        35.138s
Ran all test suites.
error Command failed with exit code 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by salsakran** [comment]
Does this occur consistently? There are still a few race conditions in our test suite we're trying to stamp out.

**Comment by jonatasfreitasv** [comment]
Yes, every time.

I'm asking so many questions and opening issues, because I want to become a Metabase contributor.

What are the best materials you suggested to read?

**Comment by attekei** [comment]
I can't reproduce the failure locally or in CircleCI. I think that it is related to the timezones because that test respects the timezone of test environment (see https://github.com/metabase/metabase/blob/f6c92f06fcbb44c8874b5dcd5d0176a3f307df8a/frontend/test/visualizations/components/LineAreaBarRenderer.unit.spec.js#L12). Maybe we should override the timezone for that test so that it will behave consistently wherever the tests are run – unless that is an actual timezone bug.

@jonatasfreitasv Glad to hear that you are interested in contributing! You probably have already read [Developer's Guide](https://github.com/metabase/metabase/blob/master/docs/developers-guide.md), and you can also skim through [Wiki pages](https://github.com/metabase/metabase/wiki). 

You can check the list of issues with `Help Wanted` tag to find something to work nice to work on: https://github.com/metabase/metabase/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22

**Comment by flamber** [comment]
Closing, since no reply and many tests have been modified and a lot of timezone fixes was done in 0.34
